documentation/examples/runkit-example.js

```diff
@@ -0,0 +1,4 @@
+const got = require("got");
+const ISS = "http://api.open-notify.org/iss-now.json";
+
+(await got(ISS).json()).iss_position;
```

Comment:
```suggestion
(await got(ISS).json()).iss_position;

```

Reply:
I've not included the empty line at the end of the file because on the Runkit editor there would have been an empty line.

<img width="372" alt="chrome_4FgDZvgLlq" src="https://user-images.githubusercontent.com/4963274/81699360-86109f00-9467-11ea-9f51-03cb1ce1a7c7.png">

Reply:
Do it anyway :) Just to be consistent with other files ;)

Reply:
Does the file need to be included in `files` in `package.json`?